tests/libpod/query.go

```diff
@@ -55,3 +60,27 @@ func GetRunningPodByLabel(virtCli kubecli.KubevirtClient, label, labelType, name
 		return &pods.Items[0], nil
 	}
 }
+
+func GetPodByVirtualMachineInstance(vmi *v1.VirtualMachineInstance, namespace string) (*k8sv1.Pod, error) {
```

Comment:
Unrelated to this PR, but why does this function have a `namespace` argument? A VMI with pod always lives in a namespace.

Reply:
Indeed. I'll try to remove it in a follow up.